scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -1021,8 +1021,8 @@ def _create_build_dir_link(self, links_dir_path, instance):
 
 
 def change_skip_to_error_if_integration(options, instance):
-    ''' If integration mode is on all skips on integration_platforms are treated as errors.'''
-    if options.integration and instance.platform.name in instance.testsuite.integration_platforms \
+    ''' All skips on integration_platforms are treated as errors.'''
+    if instance.platform.name in instance.testsuite.integration_platforms \
         and "quarantine" not in instance.reason.lower():
```

Comment:
Just a comment: There is `Filters.QUARANTINE` do there is no need to check string in the reason, better to check predefined filter.

Reply:
let's do this in the follow-up PR

Reply:
#62810